x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -18,13 +19,23 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
     pageSize: props.pageSize,
   });
 
-  const [visibleColumns, setVisibleColumns] = useState(props.columns.map(({ id }) => id));
+  const alertsTableConfigurationRegistry = useKibana().services.alertsTableConfigurationRegistry;
+  if (!alertsTableConfigurationRegistry.has(props.pluginId)) {
+    throw new Error(
```

Comment:
Hmm. I realize this is most likely designed for dev environments but it _could_ happen in production builds (as we have nothing explicitly avoiding that) - I'd advise against doing this, but maybe we show an error warning/banner at the top of the table indicating it's not configured properly? Or perhaps we just render the error banner? Throwing an error is most likely not the UX we want here

Reply:
I'd say something like:

`Alerts table configuration error - this table does not exist in the registry. Please register the table before attempting to display it`

I don't think we need to discuss the technical details of _how_ to resolve it, as users will see this message too. I think something along those lines will help any internal dev or user understand there is some problem and know to reach out for help.

We could workshop this with @lcawl

Reply:
As discussed offline. I added an empty state (see the PR description) for this case.